Assignment 1 added

Adithya-Jayan opened the pull request.

YashasG98 commented:
- Recheck submission for question 5, it's wrong
- The points table you arrived at is wrong, recheck (try to make sure that won+drawn+lost = 38 in every team's case, that'll ensure correctness)

Adithya-Jayan commented:
Sorry, I didn't notice that the Table was wrong. I have fixed it Now.
But I wasn't able to understand why answer 5 is wrong. I added up all the cards by all referees and also added up all the cards from the initial data and they seem to result in the same value 1268.

YashasG98 commented:
My bad, the 5th question was right.
